### DISCUSSION TITLE
v1.18.0

### DISCUSSION THREAD

**Discussion by lmakarov** [docksal/docksal#1762] [General]
# 1.18.0 (2023-03-12)

## New software versions ✨

- fin v1.112.0
- Docker Desktop v4.17.0
- Docker v20.10.23
- Docker Compose v2.15.1
- VirtualBox v6.1.42
- New image versions
  - [docksal/cli v3.3](https://github.com/docksal/service-cli/releases/tag/v3.3.0)
- Switched all stacks to PHP 8.1 (`docksal/cli:php8.1-3.3` image)

## Changes and improvements ⚙️

- macOS Ventura 13.0 compatibility
  - Adjusted NFS mount read/write buffers
- Ubuntu 22.04 compatibility
- Made `DOCKSAL_ENVIRONMENT` variable visible in custom commands and add-ons by default
- Updated Solr version to v7 for the Acquia stack
- Fixed `fin image registry <image_name>` command
- Fixed `fin share` command
- Fixed network settings in Gitpod
- Updated the list of options for `fin project create` (added Drupal 10 options, removed Drupal 8)
- Updated `xhprof` image to work with arm64/Apple Silicone
- Switched `cloudflared` service to the official `cloudflare/cloudflared` image (which now supports arm64)
- Dropped legacy xdebug v2 `remote_host` option in stack configs
- Improved fin sysinfo output (added green/red colors for PASS/FAIL tests)

## Documentation 📖

- Adding Mutagen option in [Shared Volumes](https://docs.docksal.io/core/volumes/) docs
- Updated PHP version docs
- Removed Solr 4 instructions and updated the listed default version to Solr 8
- Various small updates

## Contributors 🤓

@lmakarov, @froboy, @paulsheldrake, @shelane, @halisonfernandes, @obriat, @khaledzaidan, @nicoschi, @shaal, 
@matthiasseghers, @eloso-uk, @sean-e-dietrich, @danshumaker, @rosenstrauch, @jasonawant, @anpolimus, 

## Sponsors 🤑

@markaspot, @lpeabody, @alexander-danilenko, @andreyzb, @twfahey1, @johnoltman65, [❤️ You ❤️ ](https://github.com/sponsors/docksal)

<hr /><em>This discussion was created from the release <a href='https://github.com/docksal/docksal/releases/tag/v1.18.0'>v1.18.0</a>.</em>

**Comment by dpremo-ul** [comment]
FYI when I run `fin update` I'm left still on 1.17.0.  I have to pass the new version in the command:

`DOCKSAL_VERSION=v1.18.0 fin update`

**Reply by lmakarov** [reply]
There is a bug in the update process (#1763) which was introduces in v1.17.0. It also affects v1.18.0.
The fix is going out in v1.18.1 and will, sadly, require everyone to manually run `DOCKSAL_VERSION=v1.18.1 fin update` to get it.

**Reply by lmakarov** [reply]
Docksal [v1.18.1](https://github.com/docksal/docksal/releases/tag/v1.18.1) has been released. Run `DOCKSAL_VERSION=v1.18.1 fin update` to update.

**Reply by dpremo-ul** [reply]
After I run `DOCKSAL_VERSION=v1.18.1 fin update`, and then run `fin version`, output says fin is verstion v1.113.0 (new) but Docksal is v1.18.0 (old).

**Reply by lmakarov** [reply]
Addressed the mismatch in v1.18.2. `fin update` one more time please.
